test/module/irohad/consensus/yac/yac_gate_test.cpp

```diff
@@ -246,29 +240,21 @@ TEST_F(YacGateTest, LoadBlockWhenDifferentCommit) {
 
   // verify that block we voted for is in the cache
   ASSERT_NO_THROW({
```

Comment:
I assume, `ASSERT_NO_THROW` is no longer needed, as it was here for `SpecifiedVisitor`